wiki/User_title/en.md

```diff
@@ -22,7 +22,7 @@ The colour of a user title is the same as the user's colour, which is also usual
 
 ![Screenshot of Damnae's post sidebar on the old website](img/star-shooter.png "The \"Star Shooter\" title sits above Damnae's post count.")
 
-<!-- the bottom post on https://web.archive.org/web/20131107133826/https://osu.ppy.sh/forum/t/141240 -->
+*You could find the title at the buttom of this [old osu! forum post \(Internet Archive\)](https://web.archive.org/web/20131107133826/https://osu.ppy.sh/forum/t/141240)*
```

Comment:
this contributes nothing to the article and is only used as a reference for these who edit the page, please put the HTML comment back